Display/features/gets_sensor_data.feature

```diff
@@ -0,0 +1,15 @@
+Feature: gets data from sensors
+
+    Scenario: connects to the sensors that are connected
+        Given some sensors are physically connected to the Pressure Sensor and Airflow Sensor PI Hat
```

Comment:
I would word this test as 

- Given there are these objects that represent these sensors
- When data is read from those sensors on startup
- then the software will perform these actions when all is well

That allows you to create mock interfaces to a dummy sensor object that can either spit out the "right" values or garbage, depending on what you want to test.